classes/salesforce_mapping.php

```diff
@@ -121,38 +125,39 @@ public function create_fieldmap( $posted = array(), $wordpress_fields = array(),
 	}
 
 	/**
-	* Get one or more fieldmap rows between a WordPress and Salesforce object
-	*
-	* @param int $id
-	* @param array $conditions
-	* @param bool $reset
-	* @return $map or $mappings
-	* @throws \Exception
-	*/
+	 * Get one or more fieldmap rows between a WordPress and Salesforce object
+	 *
+	 * @param int   $id The ID of a desired mapping.
+	 * @param array $conditions Array of key=>value to match the mapping by.
+	 * @param bool  $reset Unused parameter.
+	 * @return Array $map a single mapping or $mappings, an array of mappings.
+	 * @throws \Exception
+	 */
 	public function get_fieldmaps( $id = null, $conditions = array(), $reset = false ) {
 		$table = $this->fieldmap_table;
-		if ( null !== $id ) { // get one fieldmap
+		if ( null !== $id ) { // get one fieldmap.
 			$map = $this->wpdb->get_row( 'SELECT * FROM ' . $table . ' WHERE id = ' . $id, ARRAY_A );
 			$map['salesforce_record_types_allowed'] = maybe_unserialize( $map['salesforce_record_types_allowed'] );
 			$map['fields'] = maybe_unserialize( $map['fields'] );
 			$map['sync_triggers'] = maybe_unserialize( $map['sync_triggers'] );
 			return $map;
-		} elseif ( ! empty( $conditions ) ) { // get multiple but with a limitation
+		} elseif ( ! empty( $conditions ) ) { // get multiple but with a limitation.
 			$mappings = array();
 			$record_type = '';
 
+			// Assemble the SQL.
 			if ( ! empty( $conditions ) ) {
 				$where = ' WHERE ';
 				$i = 0;
 				foreach ( $conditions as $key => $value ) {
 					if ( 'salesforce_record_type' === $key ) {
-						$record_type = $value;
+						$record_type = sanitize_text_field( $value );
 					} else {
 						$i++;
 						if ( $i > 1 ) {
 							$where .= ' AND ';
 						}
-						$where .= '`' . $key . '`' . ' = "' . $value . '"';
+						$where .= '`' . $key . '` = "' . $value . '"';
```

Comment:
it goes from using PHP to concatenate string ```'`'``` to string ```' = "'```
to just doing that in the file and saving us a `.`

This was something that https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards brought up.